36-vending-machine-monitor/README.md

```diff
@@ -0,0 +1,319 @@
+# Overview
+
+Soda vending machines, especially older models, are typically not internet connected. This project shows how to retrofit a regular soda vending machine to make it internet connected using a Notecard.
+
+By using a distance sensor placed at the top of each column of soda cans, the Soda Vending Machine Monitor App provides notification when items are vended, when a particular item is running low or empty and when the column has been restocked.
+
+## App features
+
+* The app monitors the items in one or more dispensing columns using a distance sensor placed above the column. The app detects when an item is vended, when the column is running low or empty and when it has been restocked.
+
+* Tilt sensing: the app sends an alert when the vending machine is tilted.
+
+* Low power mode: detects power failure and puts the app in low power mode until power resumes. Should the battery run low, the app goes to sleep, waking periodically to check the battery charge. The app sends an alert when mains power fails, and also when battery power is low.
+
+* The app is configurable via environment variables, with notifications about variable changes and errors sent to Notehub.
+
+## You will Need
+### Hardware
+- [Notecard](https://shop.blues.io/collections/notecard/products/note-wbna-500)
+- [Notecarrier F](https://blues.io/products/notecarrier/notecarrier-f/)
+- [Swan](https://shop.blues.io/collections/swan/products/swan)
+- USB-A to Micro-USB cable
+
+For each dispensing column:
+  * [Adafruit VL53LCD sensor breakout](https://www.adafruit.com/product/5396)
+  * [Quiic connect cable](https://www.adafruit.com/product/4401)
+  * Male-to-female jumper cable (optional when using just one sensor)
+
+For Assembly:
+* Soldering iron and solder to solder the 5-pin header to each ToF sensor
```

Comment:
> ToF

Maybe just me but I had to look up what this abbreviation was.

Reply:
I added some explanation when the sensor is first mentioned.